apps/cowswap-frontend/src/modules/tradeFormValidation/hooks/useTradeFormValidationContext.ts

```diff
@@ -34,6 +35,8 @@ export function useTradeFormValidationContext(): TradeFormValidationCommonContex
 
   const isPermitSupported = useTokenSupportsPermit(inputCurrency, tradeType)
 
+  const isInsufficientBalanceOrderAllowed = tradeType === TradeType.LIMIT_ORDER
```

Comment:
we need to check also if is partially fillable.

I tried to change and there's 2 issues:
- 1) it Fails
- 2) It fails silently, so no error or warning message is show. I though @shoom3301 fixed this
<img width="1510" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/240fc6a7-f739-4d06-b7c6-21f24e10d9ab">

Reply:
From what was discussed, fill or kills should also be supported

Reply:
Fill or kills now supported